daphne/src/roles_test.rs

```diff
@@ -669,7 +669,7 @@ async fn http_post_aggregate_zero_round(version: DapVersion) {
         .await;
     assert_matches!(
         t.helper.http_post_aggregate(&req).await,
-        Err(DapAbort::UnrecognizedMessage)
+        Err(DapAbort::UnrecognizedMessage { .. })
```

Comment:
More idiomatic (you will need to implement `Default` for `UnrecognizedMessage`). Here and below.
```suggestion
        Err(DapAbort::UnrecognizedMessage::default())
```

Reply:
I just looked a bit into that, is this really what we want? Right now I'm checking that the request returns any sort of `UnrecognizedMessage`, but with `default()` it would check that the request returns exactly the default `UnrecognizedMessage` (which is not the case when `detail` contains some information about the round), no? I might be misunderstanding some notation though.

Reply:
Ahh you're right, I was a bit too quick here. This is an arm of a match statement, so it doesn't make sense to compare to the default value.